Makefile

```diff
@@ -149,9 +149,9 @@ $(CODEGEN_BIN):
 
 generate-client: codegen-clientset fmt ## Generate the client
 
-codegen-clientset: $(CODEGEN_BIN) ## Generate the k8s types and clients
+codegen-clientset: ## Generate the k8s types and clients
```

Comment:
I guess the actual _CODE_GEN_ rule can be removed in this case as well:

```
CODEGEN_BIN := $(GOPATH)/bin/codegen
$(CODEGEN_BIN):
	$(GO_NOMOD) get github.com/jenkins-x/jx/cmd/codegen
```

+1 for getting rid of the jx codegen.

Reply:
Yup, removing now.